sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -169,25 +169,28 @@ void printArgs(Arg0 arg0, Args... args) {
 
 namespace RT = cl::sycl::detail::pi;
 
+// Workaround for build with GCC 5.5.0
+// https://gcc.gnu.org/bugzilla/show_bug.cgi?id=56480)
```

Comment:
```suggestion
// https://gcc.gnu.org/bugzilla/show_bug.cgi?id=56480
```
Could you add a short description of the problem and solution in the comments, so reader will not have to go to the link in order to understand what does it mean?